src/frontend/js/user/sagas.js

```diff
@@ -1,6 +1,8 @@
 import { routerActions } from 'react-router-redux';
-import { takeLatest } from 'redux-saga';
-import { call, put } from 'redux-saga/effects';
+import { takeEvery, takeLatest } from 'redux-saga';
+import { call, put, select } from 'redux-saga/effects';
+import decodeJwt from 'jwt-decode';
```

Comment:
Not sure this library is required as payload is already decoded by `sign` or `verify` apiRoutes. Anyway keep it for now.